src/applications/claims-status/utils/helpers.js

```diff
@@ -1102,13 +1102,18 @@ export const generateClaimTitle = (claim, placement, tab) => {
     case 'detail':
       return `Your ${baseClaimTitle}`;
     case 'breadcrumb':
-      return `${tab} ${joiner} your ${baseClaimTitle}`;
+      if (claimAvailable(claim)) {
+        return `${tabPrefix} your ${baseClaimTitle}`;
+      }
+      // Default message if claim fails to load.
+      return `${tabPrefix} your claim`;
```

Comment:
Previously we would show default content during load and on error; added some extra logic here to handle the error case a little better (e.g. will show `Status of your claim` instead of using the default claim type and showing `Status of your disability compensation claim`), in the same way we already handle the document title.

We shouldn't need to worry about the list card title or detail page heading because they're only ever generated once the claim has loaded.